package.json

```diff
@@ -39,6 +39,7 @@
     "emoji-datasource": "6.1.1",
     "emoji-datasource-apple": "6.0.1",
     "emoji-regex": "10.2.1",
+    "eslint-plugin-mattermost": "github:mattermost/eslint-plugin-mattermost#5b0c972eacf19286e4c66221b39113bf8728a99e",
```

Comment:
Could you move this back to being a dev dependency? Strictly speaking, it doesn't matter for the web app (it would only really have an effect if we uploaded mattermost-webapp as an NPM package), but I think Security Team had some process around new non-dev dependencies

Reply:
I'm not sure how this happened tbh. Fixed